src/GovukNotify/Interfaces/IAsyncNotificationClient.cs

```diff
@@ -1,32 +1,33 @@
 ﻿using Notify.Models;
 using Notify.Models.Responses;
 using System.Collections.Generic;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Notify.Interfaces
 {
     public interface IAsyncNotificationClient : IBaseClient
     {
-        Task<TemplatePreviewResponse> GenerateTemplatePreviewAsync(string templateId, Dictionary<string, dynamic> personalisation = null);
+        Task<TemplatePreviewResponse> GenerateTemplatePreviewAsync(string templateId, Dictionary<string, dynamic> personalisation = null, CancellationToken cancellationToken = default);
```

Comment:
I would use `CancellationToken cancellationToken = null` it's the same thing as default but it is easier for the reader as they don't need to work out what the default value of CancellationToken is.